mlperf_logging/compliance_checker/training_4.1.0/open_common.yaml

```diff
@@ -2,5 +2,5 @@
 - KEY:
     NAME:  submission_benchmark
     REQ:   EXACTLY_ONE
-    CHECK: " v['value'] in ['resnet', 'ssd', 'stable_diffusion', 'maskrcnn', 'gpt3', 'dlrm_dcnv2', 'bert', 'rnnt', 'unet3d', 'gnn'] "
+    CHECK: " v['value'] in ['retinanet', 'stable_diffusion', 'dlrm_dcnv2', 'unet3d', 'gnn', 'llama2_70b_lora'] "
```

Comment:
bert is still there in v5.0

Reply:
This list was for v4.1, so it shouldn't have been updated. I will revert that to the original